exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -7,206 +7,114 @@ namespace TriviaGame
 {
     public class Game
     {
-        List<string> players = new List<string>();
+        private const int MIN_PLAYER_COUNT = 2;
 
-        int[] places = new int[6];
-        int[] purses = new int[6];
+        private readonly List<Player> _players = new List<Player>();
+        
+        private readonly Board _board;
+        private int _currentPlayerIndex;
+        private bool _isGettingOutOfPenaltyBox;
 
-        bool[] inPenaltyBox = new bool[6];
-
-        LinkedList<string> popQuestions = new LinkedList<string>();
-        LinkedList<string> scienceQuestions = new LinkedList<string>();
-        LinkedList<string> sportsQuestions = new LinkedList<string>();
-        LinkedList<string> rockQuestions = new LinkedList<string>();
-
-        int currentPlayer = 0;
-        bool isGettingOutOfPenaltyBox;
-
-        public Game()
+        public Game(Board board)
```

Comment:
This board instance feels like it could be created in the constructor instead of being passed through the parameter